integration_tests/__tests__/dlctests.spec.ts

```diff
@@ -1,11 +1,11 @@
-import * as CfdUtils from "../../wrap_js/cfd_utils";
 import * as CfdDlcUtils from "../../wrap_js/cfddlc_utils";
-import { DlcWalletHelper } from "./wallet_helper";
+import * as CfdUtils from "../../wrap_js/cfd_utils";
 import { DlcTestHelper } from "./test_helper";
+import { DlcWalletHelper } from "./wallet_helper";
 
-const SyncTimeout = 5000;
+const SyncTimeout = 15000;
 
-jest.setTimeout(30000);
+jest.setTimeout(100000);
```

Comment:
Is it still necessary to increase the timeout with the latest wallet testing library version?

Reply:
only global timeout need a small increase.
changed